doc/classes/AudioEffectCompressor.xml

```diff
@@ -110,18 +118,25 @@
 	</methods>
 	<members>
 		<member name="attack_us" type="float" setter="set_attack_us" getter="get_attack_us">
+			Compressor's reaction time when the signal exceeds the threshold. Value can range from 20 to 2000. Default value: [code]20ms[/code].
 		</member>
 		<member name="gain" type="float" setter="set_gain" getter="get_gain">
+			Gain applied to the output signal.
 		</member>
 		<member name="mix" type="float" setter="set_mix" getter="get_mix">
+			Balance between original signal and effect signal. Value can range from 0 (totally dry) to 1 (totally wet). Default value: [code]1[/code].
 		</member>
 		<member name="ratio" type="float" setter="set_ratio" getter="get_ratio">
+			Amount of compression applied to the audio once it passes the threshold level. The higher the ratio the more the loud parts of the audio will be compressed. Value can range from 1 to 48. Default value: [code]4[/code].
 		</member>
 		<member name="release_ms" type="float" setter="set_release_ms" getter="get_release_ms">
+			Compressor's delay time to stop reducing the signal after the signal level falls below the threshold. Value can range from 20 to 2000. Default value: [code]250ms[/code].
 		</member>
 		<member name="sidechain" type="String" setter="set_sidechain" getter="get_sidechain">
+			Reduce the sound level use another audio bus. for threshold detection.
```

Comment:
I think something got lost on this one.

Reply:
I think the first phrase was OK.
"Reduce the sound level using another audio bus for threshold detection."
Meens compression of bus1 is based on peak detection of bus2